package.json

```diff
@@ -36,8 +36,11 @@
   "module": "./esm/index.js",
   "type": "module",
   "exports": {
-    "import": "./esm/index.js",
-    "default": "./cjs/index.js"
+    ".": {
```

Comment:
I'm OK with the package.json, but I'm not sure this is needed ... would rollup-plugin-svelte fail *not* having `"."` ? Also package.json is always at the root of a module, this is a funny gotcha they have there, and all my modules are exported the same.

Reply:
The `"."` is used when importing like `require('plain-tag')` and `"./package.json"` when `require('plain-tag/package.json')` is used. I think both are needed if an additional [subpath entry](https://nodejs.org/api/esm.html#esm_subpath_exports) is exported.

Reply:
Fair enough